java/graalvm.lua

```diff
@@ -0,0 +1,124 @@
+
+--- Common libraries provided by VersionFox (optional)
+local http = require("http")
+
+--- The following two parameters are injected by VersionFox at runtime
+--- Operating system type at runtime (Windows, Linux, Darwin)
+OS_TYPE = ""
+--- Operating system architecture at runtime (amd64, arm64, etc.)
+ARCH_TYPE = ""
+
+AvailableVersionsUrl = "https://www.graalvm.org/downloads/"
+DownloadInfoUrl = "https://download.oracle.com/graalvm/%s/latest/graalvm-jdk-%s_%s-%s_bin.%s"
+
+PLUGIN = {
+    --- Plugin name
+    name = "graalvm",
+    --- Plugin author
+    author = "ahai",
+    --- Plugin version
+    version = "0.0.1",
+    --- Plugin description
+    description = "graalvm JDK",
+    -- Update URL
+    updateUrl = "https://raw.githubusercontent.com/version-fox/version-fox-plugins/main/java/graalvm.lua",
+    -- minimum compatible vfox version
+    minRuntimeVersion = "0.2.2",
```

Comment:
```suggestion
    minRuntimeVersion = "0.2.4",
```

The graalvm tar.gz package has a soft-link disorder problem, which has been solved by vfox 0.2.4.